src/tags/core.js

```diff
@@ -17,6 +17,16 @@ function intStringify(node, radix, prefix) {
   return stringifyNumber(node)
 }
 
+function stringifyBool(node) {
+  const { value } = node
+  let { sourceStr } = node
+  if (sourceStr) {
+    const match = sourceStr.match(/^(?:[Tt]rue|TRUE|[Ff]alse|FALSE)$/)
+    if (!match) sourceStr = null
+  }
+  return value ? sourceStr ?? boolOptions.trueStr : sourceStr ?? boolOptions.falseStr
```

Comment:
I actually like ternaries, but five `?` on a single line is perhaps too much even for me.

I'm also not at all sure that this is right. What happens if a node is parsed with a `true` value, updated to `false` by the user, and the doc is then re-stringified? There should be a test case for that.

Reply:
done

Reply:
I'm still seeing five `?` on a single line. Rather than setting `sourceStr = null` on line 25, invert the test and do an early `return sourceStr` from there.

Reply:
Good! done.